dowhy/causal_model.py

```diff
@@ -459,4 +460,55 @@ def summary(self, print_to_stdout=False):
             print(summary_text)
         return summary_text
 
+    def refute_graph(self, k= 1, method_name =None ):
+        """
+        Check if the dependencies in input graph matches with the dataset
+        :param k: number of covariates in set Z while testing conditional independence ( X ⫫ Y ) | Z 
+        :param method_name: name of method name to test conditional independece in data
+        
+        : returns: python lists true_implications and false_implications which contain the implications which were 
```

Comment:
other refutation methods return a `CausalRefutation` object. It will be good if this method is also consistent with it. 
One problem is that the causalrefutation class has attributes like `est` which were specific to a causal estimate. How about inheriting that class and creating a new class, GraphRefutation?

In that case, you can have attributes that provide full knowledge about the test. Instead of true or false implications, how about: 

1. boolean to indicate whether the refutation test passes or not (i.e., only if false implications are empy set)
2. dictionary of conditional independencies checked, with value as p-value of test and boolean of passed (independence satisfied) or not.